### PR TITLE
add negative test for empty item name

### LINKED ISSUE #233: TC_01.001.02| New Item > Creatе a new item > Validate empty item name
ID: TC_01.001.02 | New Item > Creatе a new item  > Validate empty item name

- Preconditions:
User is  on the New Item page

- Description:

Verify that  empty item name is not allowed

-  Steps:

1. Open New Item page.
2. Leave Item Name field empty.
3. Select any item type.
4. Observe the validation result.

- Expected Result:

1. Warning message that  field cannot be empty
2. OK button is disabled

- Comments / Notes:

Observe the validation result.

- Acceptance Criteria:

If the item name field is blank, a warning message appears indicating the name is invalid, and the "OK" button must remain disabled.

### PR DESCRIPTION
**Pull request by Miguel19774**
add negative test for empty item name

### CODE REVIEW

File: tests/test_empty_name_warning.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from selenium.webdriver.common.by import By
+
+
+def test_empty_item_mame(browser):
```
Comment (otravaster) [inline tests/test_empty_name_warning.py]: Опечатка в имени теста
test_empty_item_mame → лучше переименовать в test_empty_item_name

File: tests/test_empty_name_warning.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    element_button = browser.find_element(By.XPATH, '//a[contains(., "New Item")]')
+    element_button.click()
+    assert "/view/all/newJob" in browser.current_url
+    element_input = browser.find_element(By.ID,"name")
+
+    folder = browser.find_element(By.XPATH,"//li[contains(@class, 'folder_Folder')]")
+    folder.click()
+    ok_button =browser.find_element(By.ID,"ok-button")
```
Comment (otravaster) [inline tests/test_empty_name_warning.py]: пропущен пробел после знака "="
ok_button =browser.find_element(By.ID,"ok-button") → ok_button = browser.find_element(By.ID,"ok-button")

File: tests/test_empty_name_warning.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    warning_message = browser.find_element(By.XPATH,"//div[@id='itemname-required']")
+
+    assert element_input.get_attribute("value") == ""
+    assert warning_message.is_displayed()
```
Comment (otravaster) [inline tests/test_empty_name_warning.py:16]: Тут можно ещё проверить текст сообщения:
`assert warning_message.text.strip() == "» This field cannot be empty, please enter a valid name"`

Если текст может немного меняться (локаль, версия Jenkins), можно использовать in:
`assert "This field cannot be empty" in warning_message.text`

**Review by otravaster — APPROVED** [review]

**Review by Viktor-Kashchenko-1 — APPROVED** [review]
